### PR TITLE
Adds `atomic_write.cr`

### PR DESCRIPTION
**Pull request by chris-huxtable**
Reduces the opportunity for file corruption during write or append.

### CODE REVIEW

**Comment by RX14** [comment]
Needs a rebase already aparrently.

File: README.md [hunk 1]
```diff
@@ -346,6 +346,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [aasm.cr](https://github.com/veelenga/aasm.cr) - Easy to use finite state machine for Crystal classes
  * [accord](https://github.com/neovintage/accord) - Sharable validations for Crystal objects
  * [any_hash.cr](https://github.com/Sija/any_hash.cr) - Recursive Hash with better JSON::Any included
+ * [atomic_write.cr](https://github.com/chris-huxtable/atomic_write.cr)  -  Reduces the opportunity for file corruption during write or append
```
Comment (veelenga) [inline README.md]: There are two spaces before and after the dash

Comment (chris-huxtable) [reply README.md]: Fixed

**Comment by chris-huxtable** [comment]
Is there a way to re-run travis? Something appears to have upset it the last time it ran.

**Comment by RX14** [comment]
I don't think that `Reduces the opportunity for file corruption during write or append` is a particularly good description, more like `Library for replacing or overwriting files atomically`

**Comment by chris-huxtable** [comment]
@RX14 - How about now?

**Pull request merged**